src/shared/directives/conditional-show/conditional-show.directive.ts

```diff
@@ -31,18 +32,16 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
   ngAfterViewInit() {
     if (this.caseField.show_condition) {
       this.condition = new ShowCondition(this.caseField.show_condition);
```

Comment:
You may want to check with mario but he was always asking to keep those console.log statements there for future debugging.

Reply:
@andrewfolga is right. Those comments are there for a reason. Hide/show is quite tricky, and investigating issues if complicated without some good logging. That's why we 
decided to leave them there. At least, until we find a better approach. But for the time being, please put them back

Reply:
Wanted to say we should use console.debug, but in JavaScript it will just change the clour of the message. Have no counter arguments. Will put the comments back :)